test/doctool/test-doctool-html.js

```diff
@@ -95,6 +95,10 @@ const testData = [
     html: '<ol><li>fish</li><li>fish</li></ol>' +
       '<ul><li>Red fish</li><li>Blue fish</li></ul>',
   },
+  {
+    file: fixtures.path('altdocs.md'),
+    html: '<li><ahref="https://nodejs.org/docs/latest-v8.x/api/foo.html">8.x',
```

Comment:
```suggestion
    html: '<li><a href="https://nodejs.org/docs/latest-v8.x/api/foo.html">8.x',
```
The space will be stripped anyway, but preserving it here seems less confusing?

Reply:
Good point. Fixed.